drivers/hmis/app/graphql/types/hmis_schema/project.rb

```diff
@@ -47,7 +47,7 @@ def self.configuration
     hud_field :date_deleted
     field :user, HmisSchema::User, null: true
     field :active, Boolean, null: false
-    enrollments_field without_args: [:project_types]
+    enrollments_field filter_omit: [:project_types], filter_type_name: 'EnrollmentsForProject'
```

Comment:
I guess I missed this last time, I thought this was going to look like the below now that a `filter_args` argument is accepted
```
enrollments_field filter_args: { omit: [:project_types], type_name: 'EnrollmentsForProject' }
```